Gemfile.lock

```diff
@@ -650,7 +660,9 @@ DEPENDENCIES
   nokogiri (~> 1.6.8)
   oj (~> 2.17.4)
   omniauth!
+  openproject-emoji!
   openproject-translations!
+  openproject-wiki_boxes!
```

Comment:
Thank you for seeing this. I relied on rubymine's smart checkout feature. I also removed the dependency towards openproject-emoji.